mlxbf-bootctl.c

```diff
@@ -899,12 +1043,68 @@ size_t read_bootstream_to_buffer(const char *bootstream, void *buf, int buf_size
   return idx - buf;
 }
 
+/*
+ * Get the filtering information from ACPI, which has the
+ * irot-cert-update: L4 IROT cert needs to be updated.
+ * irot-cert-key: L4 IROT cert signature lookup key.
+ */
+static void get_psc_app_filter_info(void)
+{
+#ifndef OUTPUT_ONLY
+  char path[] = "/sys/firmware/acpi/tables/SSDT1";
```

Comment:
Question: the device name has changed from PSCM to PSC0. Is that okay if the command is executed against old UEFI/ACPI?

Reply:
The 'PSCM -> PSC0' change doesn't affect this the content of search.  This commit scan signature 'irot-cert-key' and 'irot-cert-update' within the memory block, thus not affected by old UEFI/ACPI since old UEFI/ACPI doesn't have these two entries.